Mathlib/CategoryTheory/PathCategory.lean

```diff
@@ -168,9 +166,13 @@ def composePath {X : C} : ∀ {Y : C} (_ : Path X Y), X ⟶ Y
   | _, .cons p e => composePath p ≫ e
 #align category_theory.compose_path CategoryTheory.composePath
 
+-- porting note: TODO: lint complains that this is not in simp normal form,
+-- so I commented it.
```

Comment:
What does it say the simp-normal form is?

Reply:
```
-- Found 1 error in 30 declarations (plus 98 automatically generated ones) in the current file with 13 linters

/- The `simpNF` linter reports:
SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%20form -/
#check @CategoryTheory.composePath_toPath /- simp can prove this:
  by simp only [@CategoryTheory.composePath, @CategoryTheory.Category.id_comp]
One of the lemmas above could be a duplicate.
If that's not the case try reordering lemmas or adding @[priority].
 -/
```

Reply:
This is [lean4#2042](https://github.com/leanprover/lean4/issues/2042)